cspell.json

```diff
@@ -0,0 +1,17 @@
+{
+  "dictionaries": ["custom-words"],
+  "dictionaryDefinitions": [
+    {
+      "name": "custom-words",
+      "path": "./cspell-wordlist.txt",
+      "addWords": true
+    }
+  ],
+
+  "ignoreRegExpList": [
+    "/`[a-zA-Z-_., /*']+`/g"
```

Comment:
together with `"CStyleComment"` in the `"includeRegExpList"` this excludes things surrounding with backticks from spellchecking, letting us reference variable or function names which don't spellcheck in comments (you can see a few examples where I put backticks around some words that weren't previously backtick-ed, this is why)